src/client.rs

```diff
@@ -2085,4 +2105,35 @@ mod test {
             .unwrap();
         assert_eq!(manifest.config.media_type, manifest::WASM_CONFIG_MEDIA_TYPE);
     }
+
+    #[tokio::test]
+    #[ignore]
```

Comment:
Unrelated with this PR, I think we should have a way to run this unit test (and the other ignored ones) in automated fashion.

Reply:
Agree - one of the ignored tests is failing prior to this.

I'll take a look at using testcontainer-rs for the ignored tests separately.

Reply:
I raised https://github.com/krustlet/oci-distribution/pull/24 to address this.